package.json

```diff
@@ -7,13 +7,13 @@
     "build": "vitepress build docs"
   },
   "devDependencies": {
-    "@warp-ds/component-classes": "1.0.0-alpha.116",
-    "@warp-ds/elements": "1.0.0-alpha.27",
-    "@warp-ds/uno": "1.0.0-alpha.49",
-    "@warp-ds/vue": "^1.0.0-alpha.43",
+    "@warp-ds/css": "^1.0.0",
+    "@warp-ds/elements": "1.0.0",
+    "@warp-ds/uno": "1.0.0",
+    "@warp-ds/vue": "^1.0.0",
```

Comment:
Should we have `^` for all warp packages? To make sure we get newest changes up to a new major version? 🤔

Reply:
Good catch! Fixed in [9e6f645](https://github.com/warp-ds/tech-docs/pull/96/commits/9e6f64599adda2460766a8cb11d8f0880856d3a9)